Source/Core/Common/SettingsHandler.cpp

```diff
@@ -11,9 +11,10 @@
 #include <string>
 
 #ifdef _WIN32
+#include <windows.h> //NOLINT
+// The following Windows headers must be included AFTER windows.h.
```

Comment:
Because otherwise you get 'DWORD undefined' and the like. Should I really specify that?